odoo/addons/base/tests/test_image.py

```diff
@@ -177,6 +177,19 @@ def test_13_image_process_quality(self):
         res = tools.image_process(self.img_1920x1080_jpeg)
         self.assertLessEqual(len(res), len(self.img_1920x1080_jpeg))
 
+        # CASE: JPEG optimize + bigger size => original
+        image = Image.new('RGB', (1920, 1080), color=self.bg_color)
+        # Drawing non trivial content so that optimization matters.
+        ImageDraw.Draw(image).ellipse(xy=[
+            (400, 0),
+            (1500, 1080)
+        ], fill=self.fill_color, outline=(240, 25, 40), width=10)
+        image = tools.image_apply_opt(image, 'JPEG')
```

Comment:
Is that mandatory to change the output format before the call of image_process ? Can't you use the `output_format` param of image_process ?

Reply:
The first `image` is a `PIL.Image.Image` which gets turnes into `bytes` with `image_apply_opt`, it is therefore a mandatory step. I will use distinct variables to make it more clear.